ToNote/Controls/TodoControl.cs

```diff
@@ -102,5 +136,58 @@ public void TrackKeyword(string keyword, Action action)
             else
                 extendedRTB?.TrackKeyword(keyword, action);
         }
+
+        public static readonly DependencyProperty ListTodosProperty = DependencyProperty.Register("ListTodo",
+           typeof(Todo), typeof(TodoControl), new FrameworkPropertyMetadata(null)
+           { PropertyChangedCallback = (s, e) =>
```

Comment:
The degradation of formatting across these changes is a gruesome representation of a man losing all hope and descending into madness